app/assets/javascripts/govuk_publishing_components/components/feedback.js

```diff
@@ -142,6 +167,28 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       '<h2>Sorry, we’re unable to send your message as you haven’t given us any information.</h2>',
       ' <p>Please tell us what you were doing or what went wrong</p>'
     ].join('')
+
+    this.timer = 0
+    this.pastes = 0
+    this.keypresses = 0
+
+    this.timerHoneyPot = document.createElement('input')
+    this.timerHoneyPot.setAttribute('type', 'hidden')
+    this.timerHoneyPot.setAttribute('name', 'timer')
+    this.timerHoneyPot.setAttribute('value', this.timer)
+    this.somethingIsWrongForm.appendChild(this.timerHoneyPot)
+
+    this.pastesHoneyPot = document.createElement('input')
+    this.pastesHoneyPot.setAttribute('type', 'hidden')
+    this.pastesHoneyPot.setAttribute('name', 'pastes')
+    this.pastesHoneyPot.setAttribute('value', this.pastes)
+    this.somethingIsWrongForm.appendChild(this.pastesHoneyPot)
+
+    this.keypressHoneyPot = document.createElement('input')
+    this.keypressHoneyPot.setAttribute('type', 'hidden')
+    this.keypressHoneyPot.setAttribute('name', 'keypresses')
+    this.keypressHoneyPot.setAttribute('value', this.keypresses)
+    this.somethingIsWrongForm.appendChild(this.keypressHoneyPot)
```

Comment:
Lines 175 to 191 seem to be doing the same thing three times with slight variations. Could you also do something like

```
function createHoneyPotElement(reference, name, value) {
  reference = document.createElement('input')
  reference.setAttribute('type', 'hidden')
  reference.setAttribute('name', name)
  reference.setAttribute('value', value)
  this.somethingIsWrongForm.appendChild(reference)
}

createHoneyPotElement(this.timerHoneyPot, 'timer', this.timer)
createHoneyPotElement(this.timerHoneyPot, 'pastes', this.pastes)
createHoneyPotElement(this.timerHoneyPot, 'keypresses', this.keypresses)
```

(might need `this.timer` (etc.) to be defined before calling, but might not)

This is a non-blocking issue though - the code works fine as it is.